openapi/propolis-server.json

```diff
@@ -913,11 +913,30 @@
           "silo_id": {
             "type": "string",
             "format": "uuid"
+          },
```

Comment:
Need to generate an updated doc for falcon as well

Reply:
Yeah, thank you. That was done in ec187cd.